drivers/include/shtc1.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2017, RWTH Aachen. All rights reserved.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup	drivers_shtc1
+ * @ingroup	drivers_sensors
+ * @name	Device driver interface for the SHTC1 Temperature and humidity sensor
+ * @{
+ *
+ * @file
+ * @brief	Device driver interface for the SHTC1 Temperature and humidity sensor
+ *
+ * @author	Steffen Robertz <steffen.robertz@rwth-aachen.de>
+ * @author	Josua Arndt <steffen.robertz@rwth-aachen.de>
+ */
+
+#ifndef SHTC1_H
+#define SHTC1_H
+
+#include <stdint.h>
+#include "periph/i2c.h"
+
+#ifdef __cplusplus 
+extern "C" {
+#endif
+
+typedef enum {
+    CRC_DISABLED = 0,
+    CRC_ENABLED
+}crc_type_t;
+
+typedef struct {
+    float temp;
+    float rel_humidity;
+}shtc1_values_t;
+
+
+/**
+ * @brief initializes the sensor and i2c
```

Comment:
Start with capital letter here and be consistent with `i2c` or `I2C` here and all following occurences